Hardware/rocketchip-zcu102.md

```diff
@@ -0,0 +1,177 @@
+---
+riscv_hardware: true
+cmake_plat: rocketchip-zcu102
+xcompiler_arg: -DRISCV64=1
+platform: Rocketchip
+arch: RV64IMAFDC
+virtualization: "Yes"
```

Comment:
I wonder, can't we just give this as boolean also - or does the parser fail then?
```suggestion
virtualization: true
iommu: false
```